asdf/tags/core/complex.py

```diff
@@ -1,7 +1,7 @@
 import numpy as np
 
 from asdf import util
-from asdf.types import AsdfType
+from asdf._types import AsdfType
```

Comment:
Same `AsdfType` discussion applies here.

Reply:
Same as in `asdf/reference.py`.